INSTRUCTION.md

```diff
@@ -0,0 +1,58 @@
+# INSTRUCTION.md
+
+## 1. Як запускати контейнер MySQL
+
+Щоб запустити контейнер MySQL, виконайте наступну команду в терміналі:
+
+docker run -d --name mysql-container -e MYSQL_ROOT_PASSWORD=rootpassword -e MYSQL_DATABASE=app_db -e MYSQL_USER=app_user -e MYSQL_PASSWORD=1234 -v mysql_data:/var/lib/mysql -p 3306:3306 mysql-local:1.0.0
```

Comment:
Using hardcoded passwords in the command is not secure. Consider using environment variables or Docker secrets to manage sensitive information.